packages/qwik-city/runtime/src/form-component.tsx

```diff
@@ -53,13 +53,39 @@ export const Form = <O, I>(
   key: string | null
 ) => {
   if (action) {
+    const isArrayApi = Array.isArray(onSubmit$);
+    // if you pass an array you can choose where you want action.submit in it
+    if (isArrayApi) {
+      const actionInArray = Array.isArray(onSubmit$) && onSubmit$.includes(action.submit as any);
```

Comment:
I find this a bit confusing... there's already `isArrayApi` check, not sure it's needed again for `actionInArray`, or am i missing something?

Reply:
oh yeah